### PR TITLE
docs: Create example token_storage.py

### PR DESCRIPTION
**Pull request by AquaQuokka**
Signed-off-by: AquaQuokka <101112829+AquaQuokka@users.noreply.github.com>

## Summary

No, this pull request does not fix issues, but it adds an example of storing your bot token using the python-dotenv and os modules.
## Information

- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [x] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [ ] If code changes were made then they have been tested.
- [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.

### CODE REVIEW

File: examples/token_storage.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+"""
+load_dotenv('.env')
+token = os.getenv("DISCORD_TOKEN")
+"""
+
+# Then, edit your bot.run to use the token variable, instead of your actual token.
+
+# The final result should look something like this:
+
+"""
```
Comment (JustaSqu1d) [inline examples/token_storage.py:39]:
```suggestion
```

File: examples/token_storage.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import dotenv
+from dotenv import load_dotenv
+import discord
+
+bot = discord.Bot()
+load_dotenv('.env')
+
+token = os.getenv("DISCORD_TOKEN")
+bot.run(token)
+"""
```
Comment (JustaSqu1d) [inline examples/token_storage.py:50]:
```suggestion
```

Remove comments since this is the actual code.

**Review by JustaSqu1d — COMMENTED** [review]
Since most of this document is text, could this be a page on the guide, instead?

**Review by Lulalaby — CHANGES_REQUESTED** [review]
Change this into a markdown file. That makes more sense

**Comment by Lulalaby** [comment]
> Since most of this document is text, could this be a page on the guide, instead?

Or this. Pr /guide repo please.

**Pull request closed without merging**